serverlessworkflow/modules/ROOT/pages/supporting-services/jobs-service.adoc

```diff
@@ -0,0 +1,101 @@
+
+The main goal of the Job Service is to work with only active jobs. The Job Service tracks only the jobs that are scheduled and that need to be executed. When a job reaches a final state, the job is removed from the Job Service.
+
+When configured in your environment, all the jobs information and status changes are sent to the {product_name} `Data
+Index Service`, where they can be indexed and made available by GraphQL queries.
+
+[NOTE]
+====
+Data index service and the support for jobs information will be available in future releases.
+====
+
+=== Job Service persistence
+
+An important configuration aspect of job service is the persistence mechanism, where all job information is stored in a database that makes this information durable upon service restarts and guarantees no information is lost.
+
+[#job-service-postgresql]
+==== PostgreSQL
+
+PostgreSQL is the recommended database to use with job service.
+Additionally, it provides an initialization procedure that integrates link:https://flywaydb.org[Flyway] for the database initialization. It automatically controls the database schema, in this way all tables are created by the service.
+
+In case you need to externally control the database schema, you can check the Flyway SQL scripts in link:{kogito_apps_url}/jobs-service/jobs-service-postgresql/src/main/resources/db/jobs-service[the repository] and apply them.
+
+You need to set the proper configuration parameters when starting job service.
+The example shows how to run PostgreSQL as a Kubernetes deployment, but you can run it the way it fits in your environment, the important part is to set all the configuration parameters points to your running instance of PostgreSQL.
+
+==== Ephemeral
+Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
+
+[NOTE]
+====
+The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and Ephemeral (InMemory) options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `ephemeral` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}[here].
+====
+
+== Job service leader election
+
+Currently, job service works in a single instance manner where there should be just one active instance of the service.
+
+To avoid issues when deploying the service in the cloud, where it is common to eventually have more than one instance deployed, the job service supports a leader instance election process. Only the instance that become the leader activates the external communication to receive and schedule jobs.
```

Comment:
```suggestion
To avoid issues when deploying the service in the cloud, where it is common to eventually have more than one instance deployed, the Job Service supports a leader instance election process. Only the instance that become the leader activates the external communication to receive and schedule jobs.
```